### ISSUE TITLE
Grammar: incorrect parsing of the last method call in expression

### ISSUE THREAD

**Issue by kisate** [JetBrains/kotlin-spec#71]
`val bar = 1 + foo()` is parsed as:
```
----declaration|propertyDeclaration : null
------VAL : val
------variableDeclaration|simpleIdentifier|Identifier : bar
------ASSIGNMENT : =
------expression|...|genericCallLikeComparison : null
--------infixOperation|...|additiveExpression : null
----------multiplicativeExpression|...|IntegerLiteral : 1
----------additiveOperator|ADD : +
----------multiplicativeExpression|...|Identifier : foo
--------callSuffix|valueArguments : null
----------LPAREN : (
----------RPAREN : )
----semis|EOF : <EOF>
```
with ANTLR 4.9.1 and the last version of grammar, but version from the 3ee8926fb6882c14630489b537063cbb388c71c8 gives different result, that looks like the correct one:
```
----declaration|propertyDeclaration : null
------VAL : val
------variableDeclaration|simpleIdentifier|Identifier : bar
------ASSIGNMENT : =
------expression|...|additiveExpression : null
--------multiplicativeExpression|...|IntegerLiteral : 1
--------additiveOperator|ADD : +
--------multiplicativeExpression|...|postfixUnaryExpression : null
----------primaryExpression|simpleIdentifier|Identifier : foo
----------postfixUnarySuffix|callSuffix|valueArguments : null
------------LPAREN : (
------------RPAREN : )
----semis|EOF : <EOF>
```

**Comment by belyaev-mikhail** [comment]
What grammar file exactly are you using? On current KotlinParser.g4 in develop branch everything seems to be parsed correctly

**Comment by belyaev-mikhail** [comment]
Please re-run your use-case on https://github.com/Kotlin/kotlin-spec/blob/develop/grammar/src/main/antlr/KotlinParser.g4

**Comment by kisate** [comment]
Grammar from develop branch works fine, but I used release one.

**Comment by belyaev-mikhail** [comment]
Okay, I'll leave this issue open until the fixes are moved to release.
Thank you for your report.

**Comment by belyaev-mikhail** [comment]
Fixes are there in release 1.5, so closing the issue
